client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -589,8 +590,11 @@ const mapStateToProps = (
 	const currentRoute = getCurrentRoute( state );
 	const postTypeTrashUrl = getPostTypeTrashUrl( state, postType );
 	const siteOption = isJetpackSite( state, siteId ) ? 'jetpack_frame_nonce' : 'frame_nonce';
+	const initialQueryArguments = getInitialQueryArguments( state );
 
 	let queryArgs = pickBy( {
+		// plan_upgraded required by a JP premium blocks extension (currently known as upgrade nudge).
```

Comment:
You could maybe link to the JP PR that introduced it for more context, but I'll leave that decision up to you :slightly_smiling_face:

Reply:
Good idea, although someone could get to that from the commit log right?

Reply:
p.s. added https://github.com/Automattic/jetpack/pull/13281 - there's been a couple so far, but this one seems the most recent/directly related (?)

Reply:
> Good idea, although someone could get to that from the commit log right?

Let's say I'd like to keep barriers as low as possible for people to inform themselves on the background and rationale behind an implementation decision, thus hopefully creating an environment where people make informed decisions before modifying things, or copy-pasta-ing them :grimacing: